special-pages/pages/new-tab/app/favorites/components/PragmaticDND.js

```diff
@@ -230,7 +230,7 @@ export function useItemState(url, id) {
                         },
                     );
                 },
-                getIsSticky: () => true,
+                getIsSticky: () => false,
```

Comment:
This was actually covering up a bug - when you dragged outside of the grid, it was indicating that you can still drop, but you cannot.